tests/integ/sagemaker/lineage/conftest.py

```diff
@@ -651,6 +659,8 @@ def static_endpoint_context(sagemaker_session, static_pipeline_execution_arn):
         )
         endpoint_arn = get_endpoint_arn_from_static_pipeline(sagemaker_session)
 
+    endpoint_arn = get_endpoint_arn_from_static_pipeline(sagemaker_session)
```

Comment:
Should the above `get_endpoint_arn_from_static_pipeline(sagemaker_session)` be moved out of the if block? Since the same code is running in either case with this change.

Reply:
it is needed within the block since it should return a value only after the endpoint is deployed within the block